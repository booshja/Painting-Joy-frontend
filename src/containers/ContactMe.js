// dependencies
import React from "react";
import styled from "styled-components";
// components
import { ContactForm, PageTitle } from "../components";

const StyledContactMe = styled.div`
    display: flex;
    flex-direction: column;
    align-items: center;
`;

const StyledMessage = styled.p`
    text-align: center;
    width: 90%;
    font-family: "News Cycle", sans-serif;
    font-size: 1.4rem;
    letter-spacing: 1px;
    line-height: 1.8rem;
    margin-bottom: 2rem;
`;

const ContactMe = () => {
    return (
        <StyledContactMe>
            <PageTitle>Contact Me</PageTitle>
            <StyledMessage>
                Want to inquire about a mural? Anything else you'd like to tell
                me? Send me a message below!
            </StyledMessage>
            <ContactForm />
        </StyledContactMe>
    );
};

export default ContactMe;
