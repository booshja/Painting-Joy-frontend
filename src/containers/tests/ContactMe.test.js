import { render } from "@testing-library/react";
import ContactMe from "../ContactMe";

/** Smoke Test */
it("renders without crashing", () => {
    render(<ContactMe />);
});

/** Snapshot Test */
it("matches snapshot", () => {
    const { asFragment } = render(<ContactMe />);
    expect(asFragment()).toMatchSnapshot();
});
