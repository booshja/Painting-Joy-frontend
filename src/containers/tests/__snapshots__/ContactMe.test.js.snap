// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-eWnToP fcnWwj"
  >
    <h1
      class="sc-fWWYYk cwzJaj"
    >
      Contact Me
    </h1>
    <p
      class="sc-kTCsyW dPjXRO"
    >
      Want to inquire about a mural? Anything else you'd like to tell me? Send me a message below!
    </p>
    <form
      class="sc-csTbgd eMjUXy"
    >
      <div />
      <label
        class="sc-dFRpbK kbcGVL"
        for="name"
      >
        Name:
      </label>
      <input
        class="sc-bsatvv ezNTPM"
        id="name"
        name="name"
        placeholder="Name"
        type="text"
      />
      <label
        class="sc-dFRpbK kbcGVL"
        for="email"
      >
        Email:
      </label>
      <input
        class="sc-bsatvv ezNTPM"
        id="email"
        name="email"
        placeholder="Email:"
        type="text"
      />
      <label
        class="sc-dFRpbK kbcGVL"
        for="message"
      >
        Message:
      </label>
      <textarea
        class="sc-gIvpjk dwxvzk"
        id="message"
        maxlength="200"
        name="message"
        placeholder="Message"
        rows="4"
      />
      <button
        class="sc-euEtCV kgFXAJ"
        type="submit"
      >
        Submit
      </button>
    </form>
  </div>
</DocumentFragment>
`;
